lib/Reader/readerir.cpp

```diff
@@ -723,6 +690,16 @@ void GenIR::insertIRForUnmanagedCallFrame() {
 //
 //===----------------------------------------------------------------------===//
 
+AllocaInst *GenIR::createAlloca(Type *T, Value *ArraySize, const Twine &Name) {
+  AllocaInst *AllocaInst = LLVMBuilder->CreateAlloca(T, ArraySize, Name);
+
+  if (JitContext->Options->DoInsertStatepoints && GcInfo::isGcAggregate(T)) {
+    GcFuncInfo->recordGcAggregate(AllocaInst);
+  }
```

Comment:
I'm hoping one day soon that `DoInsertStatepoints` becomes the default, so I think I'd rather not have it be prominent in the readerir.cpp code.